src/orion/core/worker/primary_algo.py

```diff
@@ -152,22 +153,6 @@ def judge(self, point, measurements):
             self.transformed_space.transform(point), measurements
         )
 
-    @property
```

Comment:
Why are they removed?

Reply:
I think it's because this implementation was moved to AlgoWrapper. I'll double-check.